### DISCUSSION TITLE
With the SDK, can I achieve the same level of integration, with Tidal, that DJ software like DJay and Serato do?

### DISCUSSION THREAD

**Discussion by johnsq** [tidal-music/discussions#77] [Q&A]
Hi,
I have had play with the SDK and read so much of these discussions but am unsure how open the SDK currently is.

What I want is to be able to integrate a desktop application (that my company publish) with Tidal, but to the same level that we see in software where like Algoriddim DJay and Serato. So basically, full search and the means to stream full tracks.

Here are a couple of questions..

1.) Algoriddim DJay and Serato are both desktop applications yet Tidal are offering only an iOS, Android and Web SDK. This begs the question, which SDK do desktop applications like DJay and Serato use? Do they even use the public one?
2.) If it is not possible to achieve the same level of integration with the public SDK that we see in software like DJay and Serato, who can I talk to about going the same route as Algoriddim and Serato. 

I would very much appreciate some guidance here.

Thanks, John

**Comment by lucas-tidal** [comment]
Hey John,

Thanks for reaching out, I'm on the TIDAL Partnerships team, do you mind emailing our Business Development contact Herman (hjohre@tidal.com) and myself (lucasclark@tidal.com)?

To kick off conversation, please provide some background information about your company and what use cases you're interested in, thanks!

**Reply by Voyag3r3** [reply]
Hey Lucas, I have a similar question about integration to an app overlay. May i direct my questions your way, providing use case info etc as directed? Thank you!
